core/lib/dal/src/models/storage_tee_proof.rs

```diff
@@ -13,6 +13,11 @@ pub struct StorageTeeProof {
     pub attestation: Option<Vec<u8>>,
 }
 
+#[derive(Debug, Clone, sqlx::FromRow)]
+pub struct StorageBatch {
```

Comment:
Bikeshedding: IMO, creating a new struct for a single field is redundant; you can use an anonymous struct produced by `sqlx::query!` macro.

Reply:
Makes sense. I [reverted it](https://github.com/matter-labs/zksync-era/pull/3342/commits/464b5d4f1a189304fc442eebe847fee2a5d714d5#diff-0bfab201a463b9e47da3be98236bdc07e5f6695b23261dfd69d0dcfa85ae5a44). ✅